Header      header
string      PWMR_name

body        Ref_pos
body        Ref_vel
body        VWMR_pos
body        VWMR_vel
body        VWMR_desired_vel

float64     PWMR_SOC
body        PWMR_pos
body        PWMR_vel
wheel       PWMR_wheel_ang_vel
body        PWMR_desired_vel
wheel       PWMR_desired_wheel_ang_vel
wheel       PWMR_wheel_torque

wheel       TD1
wheel       TD2
wheel       ESO1
wheel       ESO2
